### PR TITLE
ncu: add use cases, use long options

### PR DESCRIPTION
**Pull request by BeLi4L**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/ncu.md [hunk 1]
```diff
@@ -20,6 +20,18 @@
...
+
+`ncu --filter /react/`
+
+- List only outdated dev dependencies
+
+`ncu --dep dev`
+
 - Display help:
 
 `ncu -h`
```
Comment (gutjuri) [inline pages/common/ncu.md:37]:
```suggestion
`ncu --help`
```

**Review by gutjuri — COMMENTED** [review]
Thanks for your contribution! Usually, we prefer long options.

File: pages/common/ncu.md [hunk 2]
```diff
@@ -20,6 +20,18 @@
 
 `ncu -i`
 
+- List minor outdated dependencies:
+
+`ncu --target minor`
+
+- List outdated dependencies that match a regex:
+
+`ncu --filter {{/react/}}`
```
Comment (acuteenvy) [inline pages/common/ncu.md:29]:
```suggestion
- List outdated dependencies that match a keyword or regular expression:

`ncu --filter {{keyword|/regex/}}`
```

File: pages/common/ncu.md [hunk 3]
```diff
@@ -20,6 +20,18 @@
 
 `ncu -i`
 
+- List minor outdated dependencies:
+
+`ncu --target minor`
```
Comment (acuteenvy) [inline pages/common/ncu.md:25]:
```suggestion
- List outdated dependencies up to the highest minor version:

`ncu --target {{minor}}`
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution.

LGTM after @acuteenvy's suggestions.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**